html/components/dropdown.stories.js

```diff
@@ -75,13 +78,13 @@ export const informationalDropdown = () => {
   return `
     <a
       class="sprk-b-Link sprk-b-Link--plain sprk-u-mrs"
-      href="#nogo"
```

Comment:
Why the removal of the href?

Reply:
I think I got trigger happy -- addressed in this commit:
https://github.com/sparkdesignsystem/spark-design-system/pull/2391/commits/831877ca5930d31cc4c8902e98824d2637fe13c5